source/docs/articles/wordpress/wordpress-and-pantheon-varnish-cache.md

```diff
@@ -0,0 +1,62 @@
+---
+title: WordPress and Pantheon's Varnish Servers
+description: Detailed information on understanding the interactions of WordPress Pantheon's Varnish edge caching servers.
+category:
+  - developing
+  - WordPress
+  - cache
+  - Varnish
+keywords: wordpress, cache, caching, varnish
+---
+While WordPress Core does come with object caching, it does not include any direct user interface (UI) to control that functionality. On Pantheon, all WordPress sites include a plugin called Pantheon Cache which not only includes a simple UI for administering the site cache, it also has programatic functionality for selectively clearing the Pantheon Varnish caching for individual pages and paths. In this article, we're going to go over some of the details about how these pieces all fit together.
+
+<h4>Pantheon Cache Administration</h4>
+In order to access the Pantheon Cache administration, log into your WordPress admin and navigate to /wp-admin/options-general.php?page=pantheon-cache. Once there, you will see the following:
+
+![pantheon wordpress cache admin](/source/docs/assets/images/wp-pantheon-cache-ui.png)
+
+The options are straight forward:
```

Comment:
Suggest: The options are straight forward, for more details see [WordPress Pantheon Cache Plugin Configuration](https://pantheon.io/docs/articles/wordpress/wordpress-pantheon-cache-plugin-configuration).

Delete items 1 & 2 below, as the content is preserved in the linked doc. Issue #1132 created to preserve recommendation not to set live environment cache expiration time to 0